bundles/org.openhab.binding.mycroft/README.md

```diff
@@ -0,0 +1,123 @@
+# Mycroft Binding
+
+This binding will connect to Mycroft A.I. in order to control it or react to events by listening on the message bus.
+
+Possibilies include :
+
+- Press a button in openHAB to wake Mycroft without using a wake word.
+- Simulate a voice command to launch a skill, as if you just spoke it
+- Send some text that Mycroft will say (Using its Text To Speech service)
+- Control the music player
+- Mute or duck the sound volume of Mycroft
+- React to all the aforementioned events ...
+- ... And send/receive all other kind of messages on the message bus
+
+
+## Supported Things
+
+The only thing managed by this binding is a Mycroft instance.
+
+
+## Discovery
+
+There is no discovery service, as Mycroft doesn't announce itself on the network.
```

Comment:
That's a pity - actually, it might depend on what "instance" you actually have - the Mark I seems to announce itself through mDNS and I am pretty sure the Mark II will do the same. For Picroft, I would assume it does, but I haven't tested. What setup are you using?

Reply:
Oh, mDNS for Mark ? That's new for me !
My "instances" are Mycroft running from sources (git checkout) on several raspberrys in my house.
I search a bit but didn't find any reference to a mDNS announce. Do you have a link or something ?
I'm not very familiar with mDNS, but I used a tool to check mDNS announce on the network ; I found many things but not my Mycroft "instances".

Reply:
Pfew, now that I look for a link, I cannot find any anymore... Not sure, maybe I had only dreamt of it. 🙄